pkg/cloud/azure/services/certificates/certificates_test.go

```diff
@@ -40,12 +44,14 @@ func TestCreateOrUpdateCertificates(t *testing.T) {
 
 	certsSvc := NewService(&scope)
 
-	ctx, cancel := context.WithTimeout(context.Background(), 20*time.Second)
+	ctx, cancel := context.WithTimeout(context.Background(), 60*time.Second)
 	defer cancel()
-	if err := certsSvc.CreateOrUpdate(ctx); err != nil {
-		t.Errorf("Error creating certificates")
+	if err := certsSvc.CreateOrUpdate(ctx, nil); err != nil {
+		t.Errorf("Error creating certificates: %v", err)
```

Comment:
We should be using errors.Wrap here.

Reply:
this is for testing, but agree there are places where fmt.Errorf is used everywhere which will be eventually moved.